src/patternfly/components/LogViewer/log-viewer.scss

```diff
@@ -85,6 +96,24 @@
     word-break: break-all;
   }
 
+  &.pf-m-nowrap {
+    --pf-c-log-viewer__text--WhiteSpace: var(--pf-c-log-viewer--m-nowrap__text--WhiteSpace);
+
+    .pf-c-log-viewer__list {
+      position: absolute;
+      right: var(--pf-c-log-viewer__list-item--Right);
+      left: var(--pf-c-log-viewer--m-nowrap__list-item--Left);
+      width: var(--pf-c-log-viewer__list-item--Width);
+      overflow: auto;
+    }
+
+    .pf-c-log-viewer__index {
```

Comment:
Is it OK to apply these styles to the whole component, but not only the `pf-m-nowrap`? The index should always be fixed and set left as `0` no matter the text is wrapped or not.